qiskit_quantuminspire/qi_results.py

```diff
@@ -23,21 +23,35 @@ def process(self, job: Job) -> Result:
         """
 
         results = []
+        batch_job_success = [False] * len(self._raw_results)
 
-        for result in self._raw_results:
+        for idx, result in enumerate(self._raw_results):
+            counts = {}
+            shots = 0
+            experiment_success = False
+
+            if isinstance(result, RawJobResult):
```

Comment:
Alias RawJobResult is inconsistent with alias JobResult in qi_jobs.py

Reply:
changed function signature